SPECS/nvidia-container-toolkit/nvidia-container-toolkit.spec

```diff
@@ -94,6 +96,9 @@ rm -f %{_bindir}/nvidia-container-toolkit
 %{_bindir}/nvidia-ctk
 
 %changelog
+* Mon Jul 10 2023 Henry Li <lihl@microsoft.com> - 1.13.5-1
+- Upgrade to version 1.13.3
```

Comment:
```suggestion
- Upgrade to version 1.13.5
```

Reply:
Mention that you're patching out the errors.Join functionality in the changelog.